cypress/support/pages/PageObject.js

```
@@ -0,0 +1,9 @@
+export class PageObject {
+  visit(path) {
+    cy.visit(path);
+  }
+
+  getByQa(selector) {
+    return cy.get(`[data-qa="${selector}"]`);
```

Add a defensive check for the `selector` argument to `getByQa` so tests fail early with a clear error when a falsy selector is passed. This helps catch typos. Example (optional):

```javascript
getByQa(selector) {
  if (!selector) throw new Error('getByQa: selector is required');
  return cy.get(`[data-qa="${selector}"]`);
}
```

Current implementation is correct and will work as-is, this is just a helpful improvement.